tox.ini

```diff
@@ -32,6 +32,7 @@ usedevelop = True
 ignore_outcome =
     djangomaster: True
 commands =
+    python manage.py migrate
```

Comment:
Shouldn't the flake8 test be placed before anything else?

Reply:
You are right.